### DISCUSSION TITLE
[Dataset/Title]: High Resolution Settlement Layer (HRSL)

### DISCUSSION THREAD

**Discussion by leonazareth** [samapriya/awesome-gee-community-datasets#268] [General]
### Contact Details

leonazareth@gmail.com

### Provide Dataset Link

https://gee-community-catalog.org/projects/hrsl/

### Describe the update

I am submitting it as an update request, but I want to know if it can be a problem with one of the compared datasets.

I recently downloaded the High-Resolution Settlement Layer (HRSL) from the gee-community dataset. I compared it with the dataset I had just downloaded from the HDX portal (where the official data is supposed to be) for the [Dominican Republic](https://data.humdata.org/dataset/dominican-republic-high-resolution-population-density-maps-demographic-estimates) (file: population_dom_2018-10-01.geotiff.zipGeoTIFF). 

By comparing them, I realized that they are pretty different. Not only the values of the population in each pixel but also the data coverage. In the following images, it is possible to see the gee-community dataset in red, the HDX dataset in green, and a comparison of the value of the same pixel.

![image](https://github.com/samapriya/awesome-gee-community-datasets/assets/48924229/e3bcf5b6-5b38-4401-8b1a-f979e8f235d2)

![image](https://github.com/samapriya/awesome-gee-community-datasets/assets/48924229/e38ebd64-b24b-4ea4-b3a7-af749217966a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samapriya** [comment]
This looked more like a discussion than an issue so converted it, can you share your comparison script and the HDX assets. Unfortunately the team that built HRSL was dismanteled by Facebook so there has not been any new updates in a while

**Reply by leonazareth** [reply]
Thanks for the quick answer and for posting with the wrong tag. I was wondering if it was a problem updating.

Following is the HDX asset I use to compare:
[population_dom_2018-10-01.zip](https://github.com/user-attachments/files/16088727/population_dom_2018-10-01.zip)

The code is the following: the _var HDX_HRSL_ is the HDX asset.

```
var HRSL_general = ee.ImageCollection("projects/sat-io/open-datasets/hrsl/hrslpop");


var countries = ee.FeatureCollection("USDOS/LSIB_SIMPLE/2017")

// Load packages for basemap and palettes
var palettes = require('users/gena/packages:palettes')
var style = require('users/gena/packages:style')


print('Total images',HRSL_general.size())

//Zoom to some place in the world
Map.setCenter(-69.6576, 18.4629)
//Adding country boundaries (Feel free to remove)
Map.addLayer(ee.Image().paint(countries,0,1), {color:'00FF00'}, 'Area of Interest')


Map.addLayer(HRSL_general.median(),{palette: palettes.colorbrewer.Reds[9]},'HRSL')
Map.addLayer(HDX_HRSL,{palette: palettes.colorbrewer.PuBuGn[9]},'HRSL')


style.SetMapStyleGrey()
```

I did not know about the project's discontinuation, and it's a real shame that this kind of data is so helpful for so many things.

**Reply by samapriya** [reply]
can you share your HDX_HRSL earth engine asset too ?

**Reply by leonazareth** [reply]
This is the link for the script:
https://code.earthengine.google.com/68c7eeee07d002b961376e4235d2a7ba

Here is the link for the asset:
https://code.earthengine.google.com/?asset=projects/ee-leonazareth/assets/population_dom_2018-10-01

I have never shared a gee asset, I hope it works.

**Comment by samapriya** [comment]
Okay it seems that the file in the S3 bucket is more recent than the HDX upload so I am going to assume here that the community catalog dataset is a more current copy of the source from 2022-08-31 compared to HDX file which was uploaded 2019. I cannot be sure if the update changed anything since there are no version numbers on the HDX updates. 

You can find the dataset here which was used to sync in the past https://registry.opendata.aws/dataforgood-fb-hrsl/

**Reply by leonazareth** [reply]
Thanks for the quick answer and for the clarification @samapriya , the gee-community dataset is extremely valuable

**Reply by samapriya** [reply]
So glad it is useful @leonazareth :) closing this discussion for now
